src/gui/newwizard/navigation.cpp

```diff
@@ -0,0 +1,80 @@
+#include "navigation.h"
+
+#include <QDebug>
+#include <QRadioButton>
+
+namespace OCC::Wizard {
+
+Navigation::Navigation(QWidget *parent)
+    : QWidget(parent)
+{
+    // this class manages its own layout
+    setLayout(new QHBoxLayout(this));
+}
+
+void Navigation::setEntries(const QList<SetupWizardState> &newEntries)
+{
+    // TODO: more advanced implementation (reuse existing buttons within layout)
+    // current active page is also lost that way
+    removeAllItems();
+
+    for (const auto state : newEntries) {
+        const QString &text = Utility::enumToDisplayName(state);
```

Comment:
```suggestion
        const QString text = Utility::enumToDisplayName(state);
```
Reference to a tmp, valid in that case but not of great help